src/main/scala/io/sdkman/changelogs/SchemaCrawlerMigration.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "071")
```

Comment:
Not sure where you got that number from. Did you run the following as described on the wiki page?
```
grep -R --no-filename '@ChangeLog' | sort | tail -n 1
```

Reply:
Yes. Each time I ran it, it increased the number. I ran it on Windows, in Git Bash. Let me try to run it on Ubuntu.

Reply:
I changed the number to changeset order number to 068. Hope that works.